### PR TITLE
[MSX+] DinoMod LyleSY work

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary
DinoMod tiles by LyleSY based on older MSX+ work

#### Content of the change

Tiles with changes either automatically retinted in Photoshop or GIMP from MSX+ originals, especially from https://github.com/Theawesomeboophis/UndeadPeopleUnpacked/pull/11

Pachyrhinosaurus, rid_mon_pachyrhinosaurus, pentaceratops, rid_mon_pentaceratops, zriceratops, and styracosaurus are all based on the same triceratops base from here https://github.com/I-am-Erk/CDDA-Tilesets/pull/308

That also has the provenance for the utahraptor art that zutahraptor_shady is based on, as well as the ankylosaurus art that zankylosaurus_shady is based on. That also has the allosaurus art that mon_zallosaurus_shady is based on.

#### Testing

Looks right and builds correctly. Composes and loads no errors, looks right.

Screen Shot 2023-02-02 at 6 17 56 PM

#### Additional information

Originally created for UDP

### CODE REVIEW

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pachyrhinosaurus/mon_pachyrhinosaurus.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_pachyrhinosaurus",
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pachyrhinosaurus/mon_pachyrhinosaurus.json:2]: You don't need the json if the id and the name of the sprite are the same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pachyrhinosaurus/rid_mon_pachyrhinosaurus/rid_mon_pachyrhinosaurus.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "rid_mon_pachyrhinosaurus",
+  "fg": [
+    "rid_mon_pachyrhinosaurus"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pachyrhinosaurus/rid_mon_pachyrhinosaurus/rid_mon_pachyrhinosaurus.json:4]: same don't need json here

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pentaceratops/mon_pentaceratops.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_pentaceratops",
+  "fg": [
+    "mon_pentaceratops"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pentaceratops/mon_pentaceratops.json:4]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pentaceratops/rid_mon_pentaceratops/rid_mon_pentaceratops.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "rid_mon_pentaceratops",
+  "fg": [
+    "rid_mon_pentaceratops"
+  ],
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pentaceratops/rid_mon_pentaceratops/rid_mon_pentaceratops.json:5]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_styracosaurus/mon_styracosaurus.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_styracosaurus",
+  "fg": [
+    "mon_styracosaurus"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_styracosaurus/mon_styracosaurus.json:4]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zalbertosaurus/mon_zalbertosaurus.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_zalbertosaurus",
+  "fg": [
+    "mon_zalbertosaurus"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zalbertosaurus/mon_zalbertosaurus.json:4]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zallosaurus_shady/mon_zallosaurus_shady.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_zallosaurus_shady",
+  "fg": [
+    "mon_zallosaurus_shady"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zallosaurus_shady/mon_zallosaurus_shady.json:4]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zankylosaurus_shady/mon_zankylosaurus_shady.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_zankylosaurus_shady",
+  "fg": [
+    "mon_zankylosaurus_shady"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zankylosaurus_shady/mon_zankylosaurus_shady.json:4]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zorvosaurus/mon_zorvosaurus.json [hunk 9]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_zorvosaurus",
+  "fg": [
+    "mon_zorvosaurus"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zorvosaurus/mon_zorvosaurus.json:4]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zriceratops/mon_zriceratops.json [hunk 10]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_zriceratops",
+  "fg": [
+    "mon_zriceratops"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zriceratops/mon_zriceratops.json:4]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zutahraptor_shady/mon_zutahraptor_shady.json [hunk 11]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_zutahraptor_shady",
+  "fg": [
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zutahraptor_shady/mon_zutahraptor_shady.json:3]: same

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zyrannosaurus_shady/mon_zyrannosaurus_shady.json [hunk 12]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "mon_zyrannosaurus_shady",
+  "fg": [
+    "mon_zyrannosaurus_shady"
```
Comment (Fris0uman) [inline gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_zyrannosaurus_shady/mon_zyrannosaurus_shady.json:4]: same

**Review by Fris0uman — COMMENTED** [review]
Some of the json files are not necesary. Do you have the source for the new dinos? I know the triceratops is an edit of a wiki image but what about the others?

File: gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pachyrhinosaurus/mon_pachyrhinosaurus.json (hunk 1 above)
Comment (LyleSY) [reply gfx/MShockXotto+/pngs_large_64x80/mods/DinoMod/monster/mon_pachyrhinosaurus/mon_pachyrhinosaurus.json]: Thanks, I started without and then added to troubleshoot an unrelated issue. Good instinct, wrong direction

**Comment by Fris0uman** [comment]
I think we're missing attribution for the T-rex and spinosaur so we migt have to kick them out. For the rest it's fine since it's edit of sprite that are credited

**Comment by LyleSY** [comment]
Sniffing around in the repo, I see this is where they appeared in the tileset, added at the same time as the other Japanese wiki dinosaurs we have attribution for https://github.com/CleverRaven/Cataclysm-DDA/commit/35c317017fa86aaffefa2959016e2fd3d6a035b5#diff-36b35d40a657a1100171992e0a67b009196628636e045b968e5caaa4716df760

**Comment by LyleSY** [comment]
Looks like this is the model for T Rex, also from that same Polish amusement park https://www.123rf.com/photo_11249944_life-sized-model-of-tyrannosaurus-at-jurapark-baltow-baltow-poland.html

**Comment by LyleSY** [comment]
I'm stumped on that Spinosaurus though. I'm deleting that one for now.

**Comment by LyleSY** [comment]
I've removed those

**Comment by Fris0uman** [comment]
It's going to be more work and take longer but I think we'll make our own dinos. If we're going to use wikipedia images anyway might as well have something that fits the style of the other monsters.

**Pull request closed without merging**